File: backend-nestjs/apps/auth/src/organization/organization-user-role.entity.ts
import { Entity, Column, PrimaryColumn } from 'typeorm';

@Entity('organization_user_roles')
export class OrganizationUserRoleEntity {
  @PrimaryColumn({ name: 'org_id', type: 'bigint' })
  orgId: string;

  @PrimaryColumn({ name: 'user_id', type: 'bigint' })
  userId: string;

  @PrimaryColumn({ name: 'role_id', type: 'bigint' })
  roleId: string;

  @Column({ name: 'tenant_id', type: 'bigint' })
  tenantId: string;
}

File: backend-java/cdm-auth/src/main/java/com/cdm/auth/entity/OrganizationUserRoleEntity.java
package com.cdm.auth.entity;

import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@TableName("organization_user_roles")
@Getter @Setter @NoArgsConstructor
public class OrganizationUserRoleEntity {

    private Long orgId;
    private Long userId;
    private Long roleId;
    private Long tenantId;
}
